Source/Charts/Utils/Platform.swift

```diff
@@ -124,6 +140,26 @@ public class NSUIDisplayLink
     }
 }
 
+extension NSUIColor
+{
+    var nsuirgba: (red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat)? {
```

Comment:
though the imp is different, here are two `var nsuirgba` in one extension? typo? or two vars indeed?

Reply:
@larryonoff can you get the notification? I basically finished the review, jsut some minor issues to address.